tools/jenkins/run_jenkins.sh

```diff
@@ -117,6 +117,12 @@ elif [ "$platform" == "macos" ]
 then
   echo "building $language on MacOS"
 
+  # in some jenkins macos machine, somehow the PHP build script can't find libgrpc.dylib
+  if [ "$language" == "php" ]
+  then
+    export DYLD_LIBRARY_PATH=$(pwd)/libs/$config
```

Comment:
I think rather than here, this should be set in tools/run_tests/build_php.sh   and/or src/php/bin/run_tests.sh.

see https://github.com/grpc/grpc/blob/debian/sid/tools/run_tests/run_csharp.sh